### PR TITLE
Correct two typos and add CSV header

### PR DESCRIPTION
**Pull request by wpschaub**
Two typo fixes in the conclusion and a header with the test data as it currently looks odd to have the csv-like data at the end.

### CODE REVIEW

**Comment by wpschaub** [comment]
@wsbctechnicalblog a hotfix for the just-published post. If you are OK with the fixes, please pull the publish trigger.

File: content/agile-planning-choice.md [hunk 1]
```diff
@@ -120,12 +120,16 @@ Using the same Plans view, we can visualise the portfolio, program, and team bac
 
 # Conclusion
 
-I hope this quick glimpse into our Azure DevOps based demo project, demonstrates that you can use the **product** to visualise a variety of **processes** using the **Agile** process in a single AzDO project. There are a number of view we have not covered, such as the backlog view, taskboard, Feature timeline, and Epic Roadmap. Explore [Azure Boards](https://azure.microsoft.com/en-us/services/devops/boards/) for more details.
+I hope this quick glimpse into our Azure DevOps based demo project, demonstrates that you can use the **product** to visualise a variety of **processes** using the **Agile** process in a single AzDO project. There are a number of viewswe have not covered, such as the backlog view, taskboard, Feature timeline, and Epic Roadmap. Explore [Azure Boards](https://azure.microsoft.com/en-us/services/devops/boards/) for more details.
```
Comment (said-akram-wcbbc) [inline content/agile-planning-choice.md]: Add a space between "views" and "we"

**Review by said-akram-wcbbc — APPROVED** [review]

File: content/agile-planning-choice.md (hunk 1 above)
Comment (wpschaub) [reply content/agile-planning-choice.md]: @said-akram-wcbbc fixed.

**Comment by wpschaub** [comment]
@wsbctechnicalblog good to go!

**Review by wsbctechnicalblog — APPROVED** [review]

**Pull request merged**